connections/http.js

```diff
@@ -1,26 +1,28 @@
+/* global AbortController */
 const EventEmitter = require('events')
 const { v4: uuid } = require('uuid')
+require('@chainsafe/abort-controller/polyfill')
```

Comment:
could be a good idea to impose the latest LTS especially for security reason and like this polyfill won't be required

Reply:
Yep, that's ideal

Reply:
It would also be good to use ESM in this module so that we can use the latest `node-fetch`: 
https://github.com/node-fetch/node-fetch/issues/1279#issuecomment-915059827

Reply:
Just rewriting all with `typescript` like this we can have `typings` and `esm` at the same time - bundle with `rollup` for the tree shaking and all will be perfect.
I can work on the `TS` version if @floating is interested by

Reply:
TS across Frame would be great.

Reply:
Ok so I was in a haze yesterday when replying to this.  Electron apps use the Node version provided by the version of Electron they use, currently Frame uses Electron 15 which provides Node 16.5.0:
https://www.electronjs.org/releases/stable?version=15&page=3#15.0.0

So we should be ok to remove the polyfill because the AbortController was introduced in N15.